packages/code-du-travail-api/package.json

```diff
@@ -11,7 +11,7 @@
     "start": "node ./dist/index.js",
     "dev": "nodemon ./src/server/index.js",
     "pretest": "node tests/create_indexes.js",
-    "test": "ELASTICSEARCH_DOCUMENT_INDEX=cdtn_document_test ELASTICSEARCH_ANNUAIRE_INDEX=cdtn_annuaire_test jest",
+    "test": "ELASTICSEARCH_DOCUMENT_INDEX=cdtn_document_test ELASTICSEARCH_ANNUAIRE_INDEX=cdtn_annuaire_test ELASTICSEARCH_CONVENTION_DOCUMENT_INDEX=convention_test jest",
```

Comment:
ELASTICSEARCH_CONVENTION_INDEX should be enough :)

Reply:
done !